sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-2021-2022-awards-amount-SBSD.e2e-spec.ts

```diff
@@ -0,0 +1,106 @@
+} from "../../../test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-SBSD.`, () => {
+  it("Should determine provincialAwardSBSDAmount when awardEligibilitySBSD is true and offeringCourseLoad is 40 and up", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+
+    // Assert
+    expect(calculatedAssessment.variables.programYearTotalSBSD).toBe(90);
+    expect(
+      calculatedAssessment.variables.dmnPartTimeAwardAllowableLimits
+        .limitAwardSBSD40AndUpCourseLoadAmount,
+    ).toBe(800);
+    expect(calculatedAssessment.variables.provincialAwardNetSBSDAmount).toBe(
+      710,
+    );
+    expect(
+      calculatedAssessment.variables.finalProvincialAwardNetSBSDAmount,
+    ).toBe(calculatedAssessment.variables.provincialAwardNetSBSDAmount);
+  });
+
+  it("Should determine provincialAwardSBSDAmount when awardEligibilitySBSD is true and offeringCourseLoad is < 40", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+    assessmentConsolidatedData.offeringCourseLoad = 39;
+
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+
+    // Assert
+    expect(calculatedAssessment.variables.programYearTotalSBSD).toBe(90);
+    expect(
+      calculatedAssessment.variables.dmnPartTimeAwardAllowableLimits
+        .limitAwardSBSDUnder40CourseLoadAmount,
+    ).toBe(400);
+    expect(calculatedAssessment.variables.provincialAwardNetSBSDAmount).toBe(
+      310,
+    );
+    expect(
+      calculatedAssessment.variables.finalProvincialAwardNetSBSDAmount,
+    ).toBe(calculatedAssessment.variables.provincialAwardNetSBSDAmount);
+  });
+
+  it("Should determine provincialAwardNetSBSDAmount as zero when awardEligibilitySBSD is true, programYearTotalSBSD (programYearTotalFullTimeSBSD + programYearTotalPartTimeSBSD) is greater than the dmnPartTimeAwardAllowableLimits for SBSD", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+    assessmentConsolidatedData.programYearTotalFullTimeSBSD = 500;
+    assessmentConsolidatedData.programYearTotalPartTimeSBSD = 600;
+
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+
+    // Assert
+    expect(calculatedAssessment.variables.awardEligibilitySBSD).toBe(true);
```

Comment:
This condition is expected to be true for all the above test scenarios as well. I am not sure if we need to assert to make sure here.

Btw it is not a blocker.